index.html

```diff
@@ -44,12 +44,11 @@ <h1>Personal Profile</h1>
           </div>
 
           <div class="sectionContent">
```

Comment:
Nit - it doesn't really matter for published version, but css classes should be hyphenated instead of camel cased if this is just a static site